src/applications/appeals/995/components/ContactInfo.jsx

```diff
@@ -180,9 +182,7 @@ const ContactInfo = ({
 
       <Headers className={headerClassNames}>Mailing address</Headers>
       {showSuccessAlert('address', 'Mailing address')}
-      <div className="vads-u-display--flex">
-        <AddressView data={address} />
-      </div>
+      <AddressView data={address} />
```

Comment:
Should address also have `className="dd-privacy-hidden"`?

Reply:
[`AddressView` changes](https://github.com/department-of-veterans-affairs/vets-website/pull/25023/files#diff-679971a07049e21f4f9adac8d27981dd9ff871812222f63c7562ec49471cd912). I didn't include the `country` since I figured that's far enough away from PII and it might help us troubleshoot any issues we might have with international addresses.